console-extensions.json

```diff
@@ -114,5 +114,29 @@
     "flags": {
       "required": ["CLUSTER_TEMPLATES_FLAG"]
     }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "path": ["/k8s/cluster/clustertemplate.openshift.io~v1alpha1~ClusterTemplate/:name/edit"],
+      "component": { "$codeRef": "editTemplate.default" }
+    },
+    "flags": {
+      "required": ["CLUSTER_TEMPLATES_FLAG"]
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "admin-cluster-templates-console",
+      "perspective": "admin",
+      "section": "compute",
+      "name": "%plugin__clustertemplates-plugin~Cluster templates%",
+      "href": "/k8s/cluster/clustertemplate.openshift.io~v1alpha1~ClusterTemplate",
+      "insertAfter": "nodes"
+    },
+    "flags": {
+      "required": ["CLUSTER_TEMPLATES_FLAG"]
+    }
```

Comment:
This should not be included but let's keep it for now for a simpler dev experience :)

Reply:
I'll make sure to remember to remove it before it's released :)